src/components/Form/Form.tsx

```diff
@@ -0,0 +1,17 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { FormHTMLAttributes, ReactNode } from 'react';
+
+export type FormProps = {
+    children?: ReactNode;
+    'data-test-id'?: string;
```

Comment:
```suggestion
```
Included in `HTMLAttributes<HTMLFormElement>`

Reply:
Actually it's not included, the aria attributes are but not this one.

Reply:
👌